src/tests/ftest/util/server_utils.py

```diff
@@ -183,7 +183,7 @@ def __init__(self):
             #           - IO_STAT_PERIOD=10
             self.targets = BasicParameter(None, 8)
             self.first_core = BasicParameter(None, 0)
-            self.nr_xs_helpers = BasicParameter(None, 2)
+            self.nr_xs_helpers = BasicParameter(None, 8)
```

Comment:
suggest this be set to 16 consistent with other review comments. Particularly here since it controls number of xstreams per server in functional testing.